charts/sequencer/files/cometbft/config/genesis.json

```
@@ -41,13 +41,32 @@
       {{- if $index }},{{- end }}
       {{ include "sequencer.address" $value }}
       {{- end }}
-    ]
+    ],
+    "market_map": {
+      "market_map": {
+        "markets": {}
+      },
+      "last_updated": 0,
+      "params": {
+        "market_authorities": [],
+        "admin": {
+          "bech32m": "astria1rsxyjrcm255ds9euthjx6yc3vrjt9sxrm9cfgm"
```

Can maybe use templated value for admin address, also maybe move to `.Values.global.dev` else for breaking changes.
```suggestion
          "bech32m": {{ include "sequencer.address" .Values.genesis.authoritySudoAddress }}
```